Client/Frontend/Widgets/SimpleHighlightCell.swift

```diff
@@ -21,30 +21,12 @@ struct SimpleHighlightCellUX {
     static let PlaceholderImage = UIImage(named: "defaultTopSiteIcon")
     static let CornerRadius: CGFloat = 3
     static let NearestNeighbordScalingThreshold: CGFloat = 24
+    static let BorderColor = UIColor(white: 0, alpha: 0.1)
 }
 
 class SimpleHighlightCell: UITableViewCell {
-    var siteImage: UIImage? = nil {
-        didSet {
-            if let image = siteImage {
-                siteImageView.image = image
-                siteImageView.contentMode = UIViewContentMode.ScaleAspectFit
-
-                // Force nearest neighbor scaling for small favicons
-                if image.size.width < SimpleHighlightCellUX.NearestNeighbordScalingThreshold {
-                    siteImageView.layer.shouldRasterize = true
-                    siteImageView.layer.rasterizationScale = 2
-                    siteImageView.layer.minificationFilter = kCAFilterNearest
-                    siteImageView.layer.magnificationFilter = kCAFilterNearest
```

Comment:
tbh. This actually makes it look worse imo. Because the icons arent getting as large as they did in the old TopSites. I think we should just remove this.